.github/CODEOWNERS

```diff
@@ -75,6 +75,7 @@
 /avm/res/digital-twins/digital-twins-instance/ @Azure/avm-res-digitaltwins-digitaltwinsinstance-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/document-db/database-account/ @Azure/avm-res-documentdb-databaseaccount-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/res/document-db/mongo-cluster/ @Azure/avm-res-documentdb-mongocluster-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/res/elastic-san/elastic-san/ @Azure/avm-res-elasticsan-elasticsan-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
Please note, the team either does not yet exist, or is not yet approved in the hierachy of teams ([ref](https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships))

Reply:
Sorry my bad, I created the groups with prefix "@Azure" => "@Azure/avm-res-elasticsan-elasticsan-module-owners-bicep" and "@Azure/avm-res-elasticsan-elasticsan-module-contributors-bicep". I deleted those teams and created new ones: "avm-res-elasticsan-elasticsan-module-owners-bicep" and "avm-res-elasticsan-elasticsan-module-contributors-bicep".

BTW I'm seeing I was not the only one who did this mistake. I can see more teams with prefix "@azure" - see: https://github.com/orgs/Azure/teams?query=%40azure

Reply:
There's actually one more step to it. Once the teams were created, they also have to be placed in the correct hierachy of teams - most notably the reviewer & contributor groups. This will enable your team to approve subsequent PRs for this module.
If correctly requested, one of the PMs should get a noticiation and approve the new child team request in no time :)
cc: @matebarabas